spec/spec_helper.rb

```diff
@@ -1,10 +1,22 @@
 require 'nest_thermostat'
 require 'dotenv'
 require 'pry'
+require "fakeweb"
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.